src/CSI.IBTA.Administrator/Program.cs

```diff
@@ -23,6 +23,28 @@
 
 app.UseAuthorization();
 
+app.Use(async (context, next) =>
```

Comment:
This seems as a duplicate of ErrorHandlingMiddleware functionality. 
Do we need it?

Reply:
No, my bad, pushed it late at night